package.json

```diff
@@ -177,10 +177,10 @@
         "configuration": {
             "title": "HubSpot",
             "properties": {
-                "hubspot.beta": {
+                "hubspot.hublLinting": {
                     "type": "boolean",
-                    "default": false,
-                    "markdownDescription": "Enable HubSpot VS Code extension beta features\n\n*Note: beta features may require a valid `hubspot.config.yml` file*"
+                    "default": true,
```

Comment:
If someone had previously opted out of the beta, this will re-enable this feature for them. It's possible to check if they've opted out but based on a previous convo, I think we decided to just go ahead and enable for all.